test/data/landlord-immigration-check-questions-and-responses.yml

```diff
@@ -0,0 +1,50 @@
+---
+has_asylum_card:
+- 'yes'
+- 'no'
+has_documents:
+- 'yes'
+- 'no'
+has_other_documents:
+- 'yes'
+- 'no'
+immigration_application:
+- 'yes'
+- 'no'
+property:
+- 'WC2B 6SE'
+- 'B1 1PW'
```

Comment:
Yes, that's correct. I did explain that in the [commit note](7f839d1138d3e8a753b0a638f05d6ccada216df9). I'd rather not add the explanation as a comment, because it's only really relevant for this change. In particular as soon as the scenarios go away the comment would no longer make sense.